checker/src/README

```diff
@@ -10,12 +10,5 @@ http://homes.cs.washington.edu/~mernst/advice/coding-style.html
 You may also wish to see Oracle's Java code conventions:
 http://www.oracle.com/technetwork/java/javase/documentation/codeconventions-136091.html#248
 
-If you use Eclipse:
- * Apply the code formatter preferences using
-     Preferences > Java > Code Style > Formatter > Import
-   and file
-     checker-framework/checker/eclipse/checker-framework-formatter.xml
- * Apply the import order preferences using
-     Preferences > Java > Code Style > Organize Imports pref
-   and file
-     checker-framework/checker/eclipse/checker-framework.importorder
+The codebase conforms to a consistent coding style, which is enforced by
+a git pre-commit hook.
```

Comment:
Can you add a reference to `ant reformat`/`ant check-format` here? Or point to where it is described.